packages/gestalt/src/Popover/InternalPopover.js

```diff
@@ -0,0 +1,157 @@
+// @flow strict
+import { type Node, useRef, useEffect } from 'react';
+import Box from '../Box.js';
+import { useDefaultLabelContext } from '../contexts/DefaultLabelProvider.js';
+import Controller from '../Controller.js';
+import Flex from '../Flex.js';
+import InternalDismissButton from '../shared/InternalDismissButton.js';
+
+type Color = 'blue' | 'red' | 'white' | 'darkGray';
+type Size = 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'flexible' | number;
+type IdealDirection = 'up' | 'right' | 'down' | 'left';
+type Role = 'dialog' | 'listbox' | 'menu' | 'tooltip';
+
+type Props = {|
+  /**
```

Comment:
can we remove all the prop descriptions here?

we only keep the descriptions in the exposed API (unless there's a prop that we want to add more context to)